make.sh

```diff
@@ -110,5 +111,11 @@ rsync -av \
     --exclude '.git' \
     --exclude '.gitmodules' \
     --exclude 'make.sh' \
-    --exclude 'sample' \
+    --exclude './sample' \
     ./ sample/
+
+# Correct path in Dangerfile in sample
+sed -i '' "s/'.\//'.\/sample\/ios\//g" ./sample/ios/dangerfile
+sed -i '' 's/".\//".\/sample\/ios\//g' ./sample/ios/dangerfile
+sed -i '' "s/'\.swiftlint\.yml'/'\.\/sample\/ios\/\.swiftlint\.yml'/g" ./sample/ios/dangerfile
+sed -i '' "s/'Pods\/\*\*'/'sample\/ios\/Pods\/\*\*'/g" ./sample/ios/dangerfile
```

Comment:
What is the default path of a normal Dangerfile, please? At root or at `/ios`? I think you had the same issue as me when executing `uses: MeilCli/danger-action@v5` from the sample folder. 🤔 

Does `default working-directory` from the workflow help us to bypass this issue?

Reply:
To be honest, I don't want to have too much customization on the core structure, such as these path changes 🙏